itests/org.openhab.binding.mqtt.homeassistant.tests/itest.bndrun

```diff
@@ -122,12 +122,13 @@ Import-Package: \
 	com.fasterxml.jackson.core.jackson-core;version='[2.17.1,2.17.2)',\
 	com.fasterxml.jackson.core.jackson-databind;version='[2.17.1,2.17.2)',\
 	com.fasterxml.jackson.dataformat.jackson-dataformat-yaml;version='[2.17.1,2.17.2)',\
-	com.google.guava;version='[33.2.0,33.2.1)',\
 	com.google.guava.failureaccess;version='[1.0.2,1.0.3)',\
 	com.google.re2j.re2j;version='[1.2.0,1.2.1)',\
 	com.hubspot.jinjava.jinjava;version='[2.7.3,2.7.4)',\
 	javassist;version='[3.29.2,3.29.3)',\
 	org.apache.commons.commons-net;version='[3.9.0,3.9.1)',\
 	org.apache.commons.lang3;version='[3.14.0,3.14.1)',\
-	org.osgi.service.cm;version='[1.6.0,1.6.1)',\
-	org.yaml.snakeyaml;version='[2.2.0,2.2.1)'
+	org.yaml.snakeyaml;version='[2.2.0,2.2.1)',\
+	com.fasterxml.jackson.datatype.jackson-datatype-jdk8;version='[2.17.1,2.17.2)',\
```

Comment:
related to openhab/openhab-core#4399?

Reply:
I just let the resolver do its thing because it knows best.

This bundle seems to be a requirement of org.openhab.osgiify/com.hubspot.jinjava.jinjava if I look at that bundle manifest.

Without it there are also errors in the test:

```
! Failed to start bundle com.hubspot.jinjava.jinjava-2.7.3, exception Could not resolve module: com.hubspot.jinjava.jinjava [99]
  Unresolved requirement: Import-Package: com.googlecode.ipv6; resolution:="optional"
  Unresolved requirement: Import-Package: com.fasterxml.jackson.datatype.jdk8

org.osgi.framework.BundleException: Could not resolve module: com.hubspot.jinjava.jinjava [99]
  Unresolved requirement: Import-Package: com.googlecode.ipv6; resolution:="optional"
  Unresolved requirement: Import-Package: com.fasterxml.jackson.datatype.jdk8

	at org.eclipse.osgi.container.Module.start(Module.java:463)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:445)
	at aQute.launcher.Launcher.start(Launcher.java:699)
	at aQute.launcher.Launcher.startBundles(Launcher.java:679)
	at aQute.launcher.Launcher.activate(Launcher.java:585)
	at aQute.launcher.Launcher.launch(Launcher.java:403)
	at aQute.launcher.Launcher.run(Launcher.java:185)
	at aQute.launcher.Launcher.main(Launcher.java:161)
	at aQute.launcher.pre.EmbeddedLauncher.executeWithRunPath(EmbeddedLauncher.java:170)
	at aQute.launcher.pre.EmbeddedLauncher.findAndExecute(EmbeddedLauncher.java:135)
	at aQute.launcher.pre.EmbeddedLauncher.main(EmbeddedLauncher.java:52)
```

If it is not required the bundle should have an optional import for this @ccutrer. Then you can also remove the bundle from the feature.xml file

Reply:
https://mvnrepository.com/artifact/com.hubspot.jinjava/jinjava/2.7.3 lists it as a compile dependency. AFAIK that means it's not optional, no?

Reply:
It is deprecated and integrated into other Jackson modules:

> This module has become part of Jackson Java 8 Modules as of Jackson 2.8.5

https://github.com/FasterXML/jackson-datatype-jdk8

So it seems unnecessary for newer Jackson versions like 2.17.1 that OH uses.

Reply:
I had a closer look and now see the artifact still exists in newer versions but it got moved to another repo:

https://github.com/FasterXML/jackson-modules-java8/tree/master/datatypes

It seems to be used for registering the `Jdk8Module` in Jackson:

https://github.com/search?q=repo%3AHubSpot%2Fjinjava%20jdk8module&type=code

The `Jdk8Module` is used for mapping optionals to `null` values:

https://github.com/FasterXML/jackson-datatype-jdk8/blob/master/src/main/java/com/fasterxml/jackson/datatype/jdk8/Jdk8Module.java